src/main/java/Exercise15.java

```diff
@@ -0,0 +1,7 @@
+public class Exercise15 {
+    public int binaryReverse(int n) {
+        StringBuilder result = new StringBuilder(Integer.toBinaryString(n));
```

Comment:
Tìm cách xử lý trên bit

Reply:
Xử lý sao a nhỉ
Bữa em có làm cách phức tạp quá nên đổi qa cách đơn giản như này

Reply:
Bài này vốn dĩ là để thực hiện tính toán trên bit nha